frontend/src/lib/dcat-ld.js

```diff
@@ -0,0 +1,98 @@
+function findValue(element, wantedLang) {
+  let currLang = '';
+  let currVal = '';
+  for (const value of element) {
+    if (value['@language'] === wantedLang || currLang === '') {
+      currVal = value['@value'];
+      currLang = value['@language'];
+    }
+  }
+  return currVal;
+}
+
+function parseJsonLd(ld) {
+  const seenTypes = [];
+
+  for (const element of ld['@graph']) {
+    // console.log(element);
+    if (!seenTypes.includes(element['@type'])) {
+      seenTypes.push(element['@type']);
+    }
+  }
+
+  // Prototype extract dcat stuff
+
+  const dcatExports = {
+    mimeTypes: [],
+    datasets: {},
+    catalogs: {},
+    concepts: {},
+  };
+
+  const distributions = {};
+
+  // const wantedLang = 'no';
+
+  // Find distributions
+  for (const element of ld['@graph']) {
+    if (element['@type'] === 'dcat:Distribution') {
+      const desc = typeof element['dct:description'] === 'undefined'
+        ? '[Ingen beskrivelse]'
```

Comment:
Some inconsistency in choice of language (`[No Description]` vs `[Ingen beskrivelse]`)